source/2019-12-28-countdown-to-the-new-year-ember-changeset.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Countdown to The New Year - Ember Changeset
+author: Scott Newcomer
+tags: 2019, Addons, DecEmber
+alias: 2019-12-28-countdown-to-the-new-year-ember-changeset.md
+responsive: true
+---
+
+**This is the 28th in our DecEmber series–"Countdown to The New Year: 31 Days of Ember Addons". We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!**
+
+## Day 28
+
+For the 28th edition of **Countdown to The New Year** we're going to take a
+look at [`ember-changeset`](https://emberobserver.com/addons/ember-changeset).
+
+### Ember Changeset - What It Does
```

Comment:
I think for the December series title formatting, we currently use the following - what do you think?

```suggestion
### What It Does

```